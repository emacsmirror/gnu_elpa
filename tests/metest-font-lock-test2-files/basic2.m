function events=basic2(arguments)
    arguments,
        arguments(:,:) {mustBeNumeric}
    end
end
