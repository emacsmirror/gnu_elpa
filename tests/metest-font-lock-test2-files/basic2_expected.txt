kkkkkkkk vvvvvvdffffffdvvvvvvvvvd
    kkkkkkkkkd
        vvvvvvvvvttttt dtttttttttttttd
    kkk
kkk
